.gitattributes

```
@@ -0,0 +1 @@
+./Codegen/Generated/* linguist-generated=true
```

I think the `./` might be messing this up.

```suggestion
Codegen/Generated/* linguist-generated=true
```

---

I think that is correct yes.